lte/gateway/release/magma.lockfile

```diff
@@ -198,6 +198,12 @@
       "sysdep": "python3-pymemoize",
       "version": "1.0.2"
     },
+    "pyroute2": {
+      "root": true,
+      "source": "apt",
```

Comment:
the source is pypi no?

Reply:
This was auto generated. so I kept it.
`fab test package:git` is reverting it back to "apt"